### DISCUSSION TITLE
Multiple rules not allowed for a property

### DISCUSSION THREAD

**Discussion by vincentdev15** [livewire/livewire#6100] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Hello,

I have this piece of code in a livewire form component.

To reproduce the error you just have to write 2 or more rules above a property.

```
#[Rule('required', message: 'Le nom est obligatoire.', onUpdate: false)]
#[Rule('unique', message: 'Le nom existe déjà.', onUpdate: false)]
public $name = null;
```
And I get this error message : Attribute "Livewire\Attributes\Rule" must not be repeated.

### Additional information

But in the documentation it is specified that it's recommended to separate all rules.

If you have multiple rules to customize the message for, it is recommended that you use entirely separate #[Rule] attributes for each, like so ...

What happens ? Is it a bug ?

V

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment]
Please make sure that you use the latest version. It's fixed since https://github.com/livewire/livewire/pull/5876

**Comment by joshhanley** [comment]
@vincentdev15 looks like there was further issues with it, but it hopefully should now be fixed in PR #6127 and will be in the next release (beta.6).
